src/components/QueryOutput/QueryOutput.js

```diff
@@ -923,7 +923,11 @@ export class QueryOutput extends React.Component {
           response = this.getNewResponseWithCustomColumns(response, this.state.customColumns)
         }
       } catch (error) {
-        response = error
+        if (error?.data?.message === REQUEST_CANCELLED_ERROR) {
```

Comment:
It would be good to create a `handleError` function for this logic, as i see it is used below as well.